docs/docs/responses.mdx

```diff
@@ -102,6 +101,25 @@ responses:
 In this example, when `utter_greet` gets predicted as the next action, Rasa will randomly pick one 
 of the two response variations to use.
 
+#### IDs for Responses
```

Comment:
This entire section needs to go under L46 🙏🏻

Reply:
Why, when response variations are only introduced at L91?
It does not make sense to provide example with multiple responses before we defined them.